build/import/Versions.props

```diff
@@ -15,9 +15,12 @@
 
     <!-- Toolset -->
     <RoslynToolsRepoToolsetVersion>1.0.0-beta-62705-01</RoslynToolsRepoToolsetVersion>
+    <RoslynToolsModifyVsixManifestVersion>1.0.0-beta-62327-02</RoslynToolsModifyVsixManifestVersion>
+    <RoslynToolsSignToolVersion>1.0.0-beta-62414-01</RoslynToolsSignToolVersion>
     <RoslynDependenciesProjectSystemOptimizationDataVersion>2.6.0-beta1-62205-02</RoslynDependenciesProjectSystemOptimizationDataVersion>
     <MicrosoftDotNetIBCMergeVersion>4.7.1-alpha-00001</MicrosoftDotNetIBCMergeVersion>
     <MicrosoftDiaSymReaderPdb2PdbVersion>1.1.0-beta1-62624-01</MicrosoftDiaSymReaderPdb2PdbVersion>
+    <MicroBuildPluginsSwixBuildVersion>1.0.147</MicroBuildPluginsSwixBuildVersion>
```

Comment:
Why was this pulled out to a property? I don't see any other usage of it in this PR.

Though I can't see usages of `RoslynToolsModifyVsixManifestVersion` or `RoslynToolsSignToolVersion` either.

Reply:
These are used by RoslynTools.RepoToolset and without it the project fails to build because it can't find the targets files from it.